spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -307,10 +307,6 @@ def expect_logger_msg(level, msg)
     end
 
     describe 'POST saml_callback' do
-      before(:each) do
-        allow(SAML::User).to receive(:new).and_return(saml_user)
-      end
```

Comment:
[This is redundant.](https://github.com/department-of-veterans-affairs/vets-api/blob/a58358f8e2d57b242b6d04a2891c58d268f052c9/spec/controllers/v0/sessions_controller_spec.rb#L153)